newapi/ooniapi/citizenlab.py

```diff
@@ -254,36 +254,28 @@ def sync_state(self, username):
 
             self.set_state(username, "CLEAN")
 
-    def add(self, username, cc, new_entry, comment):
-        self.sync_state(username)
-        self.pull_origin_repo()
-        log.debug("adding new entry")
-        state = self.get_state(username)
-        if state in ("PR_OPEN"):
-            raise Exception("You cannot edit files while changes are pending")
+    def update(self, username: str, cc: str, old_entry: dict, new_entry: dict, comment):
+        """Create/update/delete"""
+        # TODO: set date_added to now() on new_entry
+        ks = ("url", "category_code", "date_added", "source", "notes")
```

Comment:
You are missing the `category_description` from the `old_entry_li` so below when you check for `row == old_entry_li` it will always result in a miss.

Reply:
Updated